recipe/meta.yaml

```diff
@@ -60,16 +60,15 @@ requirements:
     - liblapack
     - llvm-openmp  # [osx]
     - zlib
-    - libmetatomic-torch >=0.1.2,<0.2.0
+    - libmetatomic-torch >=0.1.7,<0.2
     # always build against the CPU version of libtorch, we can still pick the
     # cuda one at runtime
     - libtorch * cpu*
-    - libtorch
```

Comment:
Please leave this one around, this is how conda-smithy tracks a dependency on libtorch

Reply:
This will likely need a rerender after adding it back

Reply:
weirdly didn't need one, but does break